core/src/main/java/io/questdb/cutlass/line/tcp/TableUpdateDetails.java

```diff
@@ -251,18 +252,19 @@ void releaseWriter(boolean commit) {
     }
 
     public class ThreadLocalDetails implements Closeable {
+        private static final int COLUMN_INDEX_NOT_CACHED = -2;
         private final Path path = new Path();
-        private final ObjIntHashMap<CharSequence> columnIndexByNameUtf8 = new ObjIntHashMap<>();
+        private final ObjIntHashMap<CharSequence> columnIndexByNameUtf8 = new ObjIntHashMap<>(8, 0.3, COLUMN_INDEX_NOT_CACHED);
```

Comment:
this should be `CharSequenceIntHashMap` if possible. `ObjIntHashMap` may not be able to compare `string` and `CharSequence` correctly

Reply:
replaced